ui/component/playlistCard/view.jsx

```diff
@@ -160,6 +158,13 @@ const PlaylistCardComponent = (props: PlaylistCardProps) => {
   const [bodyRef, setBodyRef] = React.useState();
   const [hasActive, setHasActive] = React.useState();
   const [scrolledPastActive, setScrolledPast] = React.useState();
+  const [backgroundImage, setBackgroundImage] = React.useState();
+
+  React.useEffect(() => {
+    if (thumbnailFromClaim) {
+      setBackgroundImage('https://thumbnails.odycdn.com/optimize/s:390:0/quality:85/plain/' + thumbnailFromClaim);
+    }
+  }, [thumbnailFromClaim]);
```

Comment:
No state or effect is required here since `thumbnailFromClaim` is a prop that will cause a render when changed and everything will re-calculate anyway. State only needed for something async.

The following should be enough, unless I missed something.
```
const backgroundImage = thumbnailFromClaim ? ... : undefined.
```